corehq/apps/reminders/models.py

```diff
@@ -1006,6 +1012,36 @@ def case_changed(self, case, now=None, schedule_changed=False, prev_definition=N
         with CriticalSection([key]):
             self._case_changed(case, now, schedule_changed, prev_definition)
 
+    def get_case_criteria_reminder_start_date(self, case, now):
+        """
+        Returns (start, spawn, used_now) where:
```

Comment:
Could you return a `namedtuple` here to make this a little more explicit? Since this doesn't just return a date, might want to name this something slightly different to show that it is returning a bunch of parameters. 
Also, should this be a public method? Looks like you are only calling it from a private method below.